doc/user/content/sql/create-views.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "CREATE VIEWS"
+description: "`CREATE VIEWS` creates views for the distinct components of a multiplex stream."
+menu:
+  main:
+    parent: 'sql'
+---
+
+{{< version-added v0.8.0 />}}
+
+`CREATE VIEWS` creates views for the distinct components of a multiplex stream. For example, it separates the replication stream of a [Postgres source](/sql/create-source/postgres/) out into views that reproduce the upstream tables that populate the stream.
+
+`CREATE VIEWS` is distinct from both the more general [`CREATE VIEW`](/sql/create-view/) command, which provides an alias for `SELECT` statements, and from [materialized views](/sql/create-materialized-view).
```

Comment:
It is possible to specify the `MATERIALIZED` too like so: `CREATE MATERIALIZED VIEWS`. Maybe we should drop the last clause to avoid implying that you can't materialize views using `CREATE VIEWS`?